### PR TITLE
Update record clearance project

### LINKED ISSUE #949: Update Project Profile: Record Clearance Project
### Overview
As a product manager, I need to make sure that all the projects that are listed on the HfLA website are up to date so that my customers(visitors) can get an accurate picture of each of the projects. 

### Project card
https://www.hackforla.org/projects/record-clearance-project

### Action Items

- [x] [INSERT TEAM NAME HERE] Team Leadership
   - [x] Please check off everything that is ok the way that it is, and if not, please provide the updated information in the comments below:
      - [x] title
      - [x] description
      - [x] image
      - [ ] leadership
         - [ ] Member Name
         - [ ] Role
         - [ ] Slack Member ID
         - [ ] Github Handle
     - [ ] links
        - [x] Github
        - [x] Slack
        - [ ] Test Site
        - [ ] Demo Site
        - [x] Site
     - [x] looking
     - [x] technologies
     - [x] location
     - [x] partner
     - [x] tools
     - [x] status
- [ ] Development
   - [ ] Edit Leadership and Test Site details (See information in the comments below)

### PR DESCRIPTION
**Pull request by Treidexy**
Fixes #949

### CODE REVIEW

**Review by trtincher — APPROVED** [review]
Looks good to me. I believe the process is to get one more reviewer to approve then it is ready for squash and merge. So you might reach out to david or josh or someone to do the final review for you.

**Comment by jbubar** [comment]
Great job Teddy, one small change
If you look at issue #853, it looks like we want our slack url's to have hackforla in it rather than app.

Could you make those changes on the same branch that you did these changes on, then push it?

File: _projects/record-clearance-project.md [hunk 1]
```diff
@@ -8,18 +8,18 @@ alt: 'Record clearance project'
...
-    role: Project Lead & Product Manager
+  - name: Gretchen Howard
+    role: Product Manager
     links:
-      slack: 'https://hackforla.slack.com/team/U01BU7LQFHB'
-      github: 'https://github.com/j100892'
-    picture: https://avatars.githubusercontent.com/j100892
-  - name: Daniel Xiao
-    role: Tech Lead
+      slack: 'https://app.slack.com/team/U01GC5HU9AB'
```
Comment (jbubar) [inline _projects/record-clearance-project.md]: Can you change the urls from 'https://app.slack.com/team/U01GC5HU9AB' to 'https://hackforla.slack.com/team/U01GC5HU9AB'. 
essentially change it from app.slack to hackforla.slack

**Review by jbubar — CHANGES_REQUESTED** [review]

**Comment by jbubar** [comment]
Nicely done Teddy!!
Lets get you a new pull request!

**Pull request merged**